packages/badge/README.md

```diff
@@ -36,12 +36,11 @@ import Badge from '@leafygreen-ui/badge';
 
 ## Properties
 
-| Prop        | Type                                                                  | Description                                                       | Default       |
-| ----------- | --------------------------------------------------------------------- | ----------------------------------------------------------------- | ------------- |
-| `variant`   | `'lightgray'`, `'darkgray'`, `'red'`, `'blue'`, `'green'`, `'yellow'` | Sets the style variant of the badge.                              | `'lightgray'` |
-| `className` | `string`                                                              | Adds a className to the class attribute                           |               |
-| `children`  | `node`                                                                | The content that will appear inside of the `<Badge />` component. |               |
-
-_Any other properties will be spread on the root element._
-
-Note: Specifying the `onClick` attribute will change the cursor style to pointer.
+| Prop        | Type                                                                        | Description                                                       | Default       |
+| ----------- | --------------------------------------------------------------------------- | ----------------------------------------------------------------- | ------------- |
+| `variant`   | `'lightgray'` \| `'darkgray'`\| `'red'`\| `'blue'`\| `'green'`\| `'yellow'` | Sets the style variant of the badge.                              | `'lightgray'` |
+| `className` | `string`                                                                    | Adds a className to the class attribute                           |               |
+| `children`  | `node`                                                                      | The content that will appear inside of the `<Badge />` component. |               |
+| ...         | `React.HTMLAttributes<HTMLDivElement>`                                      | Any other properties will be spread on the root element.          |               |
```

Comment:
Can we do colspan in markdown? It would be great if we had this note span all the columns.

Reply:
To be fair, I think the way it's laid out makes sense. The `...` represents our `...rest` spreading of the properties (the prop name). The type indicates how we type the rest parameters and the description is the description. Default isn't really applicable here. If you feel strongly, happy to look into it, but I sort of think that it makes more sense as-is

Reply:
I don't feel strongly, just trying hide irrelevant implementation details. For instance, will people see `...` and think `...rest`? I've found some engineers working with our components don't even understand the `...rest` syntax, so this probably isn't helpful.

Reply:
I couldn't figure out how to apply a colspan in markdown 🤔

Reply:
It might not be possible, but thought it was worth mentioning 😄 

FWIW you might be able to use HTML to define it (most markdown parsers support inline HTML.